README.md

```diff
@@ -20,7 +20,7 @@ you can automatically subsidize gas for transactions that match certain criteria
 
 ## Prerequisites
 
-- Python 3 (to run the `roll.py` script)
+- Python >= 3.10 (to run the `roll.py` script)
```

Comment:
Do we know if we have dependencies that require python >= 3.10?

Reply:
Yes, https://github.com/0xFableOrg/simple-op-stack-rollup/blob/master/libroll.py#L8 this syntax need >=3.10